interfaces/builtin/network_manager.go

```diff
@@ -67,6 +67,29 @@ network packet,
 
 /dev/rfkill rw,
 
+# Needed for modem connections using PPP
+/usr/sbin/pppd ix,
+/etc/ppp/** rwix,
+/dev/ppp rw,
+/dev/tty[^0-9]* rw,
+/run/udev/data/* r,
+/proc/sys/kernel/modprobe r,
```

Comment:
I am not sure if the better thing would be (if we keep ppp in the core snap) to create a ppp interface which would be implicitly provided by the core snap. However we're not doing that for wpa-supplicant as of today which is used the same way from the network-manager snap. If @zyga @jdstrand or @niemeyer don't have anything against this I am fine to keep those for now.